android/src/main/java/io/mosip/tuvali/wallet/transfer/TransferHandler.kt

```diff
@@ -123,7 +123,7 @@ class TransferHandler(looper: Looper, private val central: Central, val serviceU
     if (report.type == TransferReport.ReportType.SUCCESS) {
       currentState = States.TransferVerified
       transferListener.onResponseSent()
-      Log.d(logTag, "handleMessage: Successfully transferred vc in ${System.currentTimeMillis() - responseStartTimeInMillis}ms")
+      //Log.d(logTag, "handleMessage: Successfully transferred vc in ${System.currentTimeMillis() - responseStartTimeInMillis}ms")
```

Comment:
Make this as a info log. Do not comment this log. It gives us an idea on the transfer time.

Reply:
done